Technical-Steering-Committee/guidelines/major-release.md

```diff
@@ -0,0 +1,83 @@
+# Vnext guidelines
+
+The preparation for Vnext starts two PIs (half a year) before the actual GA date. The GA date
+is expected to be aligned with an end of a PI. 
+
+## Key Milestones
+
+Let's assume we will have release on the end of 24PI1. This would mean that the preparation for the major 
+release will start with 23PI4 and continue through 24PI1.  
+
+- PI - 1 (Approximately 180 days before the release, In an example this means 23PI4)
+- Release PI (Approximately 90 days before release, In an example this means 24PI1)
+- Day 0 represents the GA date for the release
+
+### The key events
+
+- The zowe.org/vNext contains basic information for the new major release (Start of PI - 1)
+  - Includes the dates for the Office Hours for Extenders
+- Documentation for next major version is available on staging (During the PI - 1)
+- Technical preview build is available to the general public (Start of Release PI, could be earlier)
+
+## PI - 1
```

Comment:
Add the note with the place to put the ideas for the V3 for the other squads to see.